docs/_partner_api/endpoints.md

```diff
@@ -382,6 +384,7 @@ curl -H "Authorization: Bearer {token}" -L "https://api.zapier.com/v1/profiles/m
 
 | attribute       | type   | notes                                                                                                                                                 |
 | --------------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **uuid**        | String | An unique canonical identifier to the app                                                                                                             |
```

Comment:
Small correction:  `A unique` 😄

Reply:
🤔  I wonder if it makes more sense to phrase it as `the canonical identifier of the app`?